paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -145,13 +148,17 @@ def issue_state_change_for_service(service_config, force_bounce, desired_state):
 def paasta_start_or_stop(args, desired_state):
     """Requests a change of state to start or stop given branches of a service."""
     instance = args.instance
-    cluster = args.cluster
+    clusters = args.clusters
     soa_dir = args.soa_dir
     service = figure_out_service_name(args=args, soa_dir=soa_dir)
+    if args.clusters is not None:
+        clusters = args.clusters.split(",")
+    else:
+        clusters = list_clusters(service)
 
     service_config = get_instance_config(
         service=service,
-        cluster=cluster,
+        cluster=clusters[0],
```

Comment:
Doing this means you're only going to validate that a deployment exists for the service config for the first cluster in the clusters list. Can you validate every service config instead? Something like 
```
service_configs = [get_instance_config(...) for cluster in clusters]
invalid_deploy_groups = [service_config.get_deploy_group() for service_config in service_configs if service_config.get_deploy_groups() not in remote_refs]
if invalid_deploy_groups:
    print "No branches found for %s in %s." % \
             (' ,'.join(invalid_deploy_groups), remote_refs)
          print "Has %s been deployed there yet?" % service
          return 1
```
Around line 180